requirements.txt

```diff
@@ -8,36 +10,47 @@ freezegun==0.3.11; python_version < '3.8'
 freezegun==0.3.15; python_version >= '3.8'
 gevent==1.5.0 ; python_version == '3.7'
 gevent==20.9.0 ; python_version > '3.7' and python_version <= '3.9'
-gevent==21.8.0 ; python_version > '3.9'  # (Jammy)
+gevent==21.8.0 ; python_version > '3.9' and python_version <= '3.10'  # (Jammy)
+gevent==22.10.2; python_version > '3.10'
 greenlet==0.4.15 ; python_version == '3.7'
 greenlet==0.4.17 ; python_version > '3.7' and python_version <= '3.9'
-greenlet==1.1.2 ; python_version  > '3.9'  # (Jammy)
+greenlet==1.1.2 ; python_version  > '3.9' and python_version <= '3.10'  # (Jammy)
+greenlet==2.0.2 ; python_version > '3.10'
 idna==2.8
 Jinja2==2.11.3 # min version = 2.10.1 (Focal - with security backports)
-libsass==0.18.0
-lxml==4.6.5 # min version = 4.5.0 (Focal - with security backports)
-MarkupSafe==1.1.0
+libsass==0.20.1
+lxml==4.6.5 ; python_version <= '3.10'  # min version = 4.5.0 (Focal - with security backports)
+lxml==4.9.2 ; python_version > '3.10'
+MarkupSafe==1.1.1 ; python_version <= '3.10'
+MarkupSafe==2.1.2 ; python_version > '3.10'
 num2words==0.5.6
 ofxparse==0.19; python_version <= '3.9'
 ofxparse==0.21; python_version > '3.9'  # (Jammy)
 passlib==1.7.3 # min version = 1.7.2 (Focal with security backports)
-Pillow==9.0.1  # min version = 7.0.0 (Focal with security backports)
+Pillow==9.0.1 ; python_version <= '3.10'  # min version = 7.0.0 (Focal with security backports)
+Pillow==9.4.0 ; python_version > '3.10'
 polib==1.1.0
-psutil==5.6.7 # min version = 5.5.1 (Focal with security backports)
-psycopg2==2.7.7; sys_platform != 'win32' and python_version < '3.8'
-psycopg2==2.8.6; sys_platform == 'win32' or python_version >= '3.8'
+psutil==5.8.0 ; python_version <= '3.10' 
+psutil==5.9.4 ; python_version > '3.10' 
+psycopg2==2.7.7 ; sys_platform != 'win32' and python_version < '3.8'
+psycopg2==2.8.6 ; sys_platform != 'win32' and python_version >= '3.8'and python_version <= '3.10'
+psycopg2==2.8.6 ; sys_platform == 'win32' and python_version < '3.10'
```

Comment:
@odony Finally I changed to use two lines instead of a very long and complex condition.